app/src/main/res/layout/activity_monitoring_group_page.xml

```diff
@@ -15,7 +15,30 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content">
 
+            <TableLayout
```

Comment:
![image](https://user-images.githubusercontent.com/23643192/108653659-0b17f280-7495-11eb-9d7a-b91794580668.png)
I don't think you should create a tablelayout for the button, since we are only putting one single entry in a table. Tables should be used if we want to present data in a table format (columns and rows).

![image](https://user-images.githubusercontent.com/23643192/108653706-297dee00-7495-11eb-8648-8d70e31827f2.png)
I believe the button should be within this linearlayout. You can simply add the button beside the group name.

![image](https://user-images.githubusercontent.com/23643192/108655992-dce7e200-7497-11eb-9343-0b7a0aa72fb3.png)
It could look something like this which is much cleaner.

![image](https://user-images.githubusercontent.com/23643192/108656220-eb35fe00-7497-11eb-816f-5bfcaa861082.png)
This is the code for it if you plan on doing it that way. You just need to delete the tablelayout and everything within it. You could also change the size of the button or text by adjusting the "layout_weight" property.

You don't need to do it this way; my main point is that a table shouldn't be created for a single button.

Reply:
ok, I removed the tableLayout , I placed the button  within the linearlayout beside the and group name and I ajusted it